pallets/asset/src/lib.rs

```diff
@@ -2241,4 +2280,32 @@ impl<T: Trait> Module<T> {
         );
         Ok(())
     }
+
+    /// Ensure the number of attached transfer manager extension should be < `MaxNumberOfTMExtensionForAsset`.
+    fn ensure_max_limit_for_tm_extension(
+        ext_type: &SmartExtensionType,
+        ticker: &Ticker,
+    ) -> DispatchResult {
+        if *ext_type == SmartExtensionType::TransferManager {
+            let no_of_ext = u32::try_from(
+                <Extensions<T>>::get((ticker, SmartExtensionType::TransferManager)).len(),
+            )
+            .unwrap_or_default();
+            ensure!(
+                no_of_ext < T::MaxNumberOfTMExtensionForAsset::get(),
```

Comment:
```suggestion
                no_of_ext <= T::MaxNumberOfTMExtensionForAsset::get(),
```

Reply:
We don't need `=` we are updating the extension after this check so it will automatically make it equal to `MaxNumberOfTMExtensionForAsset`.

Reply:
Please add that as a dev comment.